lib/webrtcstreamer.js

```diff
@@ -350,6 +351,7 @@ export default class WebRtcStreamer {
     pc.ondatachannel = function(evt) {
       evt.channel.onopen = function() {
         this.send('remote channel openned');
+        this.onSuccess();
```

Comment:
@AJFunk this would break, as the `this` here is the `ev.channel` and not the class ;)

So perhaps would be better to hoist a `self` a few layers up and refer that to get class methods.